test/tests/performance/pbs_preemptperformance.py

```diff
@@ -504,3 +498,102 @@ def test_preemption_basic(self):
     def tearDown(self):
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
         job_ids = self.server.select()
+        self.server.delete(id=job_ids)
```

Comment:
Put tearDown() at the end of the test suite.  Also, if you wait long enough, @latha-subramanian has a PR out right now that deletes the jobs in tearDown() and quickly.  This could hinder the test.  Also, make sure to call TestPerformance's tearDown()